lib/modules/Client.js

```diff
@@ -47,10 +48,20 @@ export default config => {
     ));
 
     ReactDOM.render(
-      <Provider store={store}>
-        {appComponent}
+      <Provider store={ store }>
+        { appComponent }
       </Provider>
       ,$container
     );
+
+    store.dispatch(actions.navigateToUrl(
```

Comment:
(Looking here from https://github.com/reddit/reddit-mobile/pull/593). Always re-routing the url might not be needed for most apps. Additionally there's no good place to do things like "pre-fetch this data when we're on the client". What would everyone think of returning `dispatch` from this thunk instead of doing the re-route? Then mobile web could do something like 
```javascript
const dispatch = Client({...})();
dispatch(shellActions.activateOnClient());
dispatch(preloadActions.preloadClientData());
``` 
`activateOnClient` could be something like: 
 ```javascript
const activateOnClient = () => async (dispatch, getState) => {
  if (!getState().shell) { return; }

  dispatch(shell.enableShell(false));
  disaptch(navigationHelpers.reRouteCurrentPage());
};
```
This would also allow `data.shell = false` in `modifyData` to be removed